tests/test_setup.py

```diff
@@ -50,12 +50,7 @@ def test_mysqlclient_tls_enforced():
 @pytest.mark.django_db
 def test_no_missing_migrations():
     """Check no model changes have been made since the last `./manage.py makemigrations`."""
-    with pytest.raises(SystemExit) as e:
-        # Replace with `check_changes=True` once we're using Django 1.10:
-        # https://code.djangoproject.com/ticket/25604
-        # https://github.com/django/django/pull/5453
-        call_command('makemigrations', interactive=False, dry_run=True, exit_code=True)
-    assert str(e.value) == '1'
+    call_command('makemigrations', interactive=False, dry_run=True, check_changes=True)
```

Comment:
The CLI name and the argument name differ in this case (helpful I know), so `check_changes` is correct (the test would fail too):
https://github.com/django/django/blob/1.10.5/django/core/management/commands/makemigrations.py#L60

Worth checking though (excuse the pun) :-)